src/main/java/com/arpnetworking/metrics/mad/Main.java

```diff
@@ -205,6 +206,9 @@ private void launchActors(final Injector injector) {
         // Create the telemetry connection actor
         actorSystem.actorOf(Props.create(Telemetry.class, injector.getInstance(MetricsFactory.class)), "telemetry");
 
+        // Create the actor sources supervisor actor
+        actorSystem.actorOf(SourceSupervisor.props(), "source");
```

Comment:
One "source" actor to supervise all source actors.  Again, this is just to group the source actors.  Each of them will start and stop independently with their config.